### PR TITLE
Add libutils3-log-dev and libspdlog-dev dependencies

### PR DESCRIPTION
**Pull request by iche033**
to go with https://github.com/gazebosim/gz-common/pull/615

### CODE REVIEW

**Review by caguero — APPROVED** [review]

File: ubuntu/debian/control [hunk 1]
```diff
@@ -12,13 +12,15 @@ Build-Depends: cmake,
                libgz-cmake4-dev,
                libgz-math8-dev,
                libgz-utils3-dev,
+               libgz-utils3-log-dev
```
Comment (caguero) [inline ubuntu/debian/control]: Missing a comma, here?

**Review by caguero — CHANGES_REQUESTED** [review]

File: ubuntu/debian/control [hunk 2]
```diff
@@ -43,7 +45,9 @@ Architecture: any
 Section: libdevel
 Depends: libgz-cmake4-dev,
          libgz-utils3-dev,
+         libgz-utils3-log-dev
```
Comment (caguero) [inline ubuntu/debian/control]: and a comma here?

**Review by caguero — CHANGES_REQUESTED** [review]

File: ubuntu/debian/control (hunk 1 above)
Comment (iche033) [reply ubuntu/debian/control]: ah yes fixed. [5ccdf78](https://github.com/gazebo-release/gz-common6-release/pull/5/commits/5ccdf78222cb9a8756cee8ac99dbfb3a4702aa4f)

File: ubuntu/debian/control (hunk 2 above)
Comment (iche033) [reply ubuntu/debian/control]: fixed. [5ccdf78](https://github.com/gazebo-release/gz-common6-release/pull/5/commits/5ccdf78222cb9a8756cee8ac99dbfb3a4702aa4f)

**Review by caguero — APPROVED** [review]

**Pull request merged**